src/bin/genesis.rs

```diff
@@ -0,0 +1,68 @@
+use dotenv::dotenv;
+use ethers::types::U256;
+use kakarot_rpc::test_utils::katana::genesis::KatanaGenesisBuilder;
+use lazy_static::lazy_static;
+use reth_primitives::B256;
+use serde::Serialize;
+use serde_with::serde_as;
+use starknet::core::serde::unsigned_field_element::UfeHex;
+use starknet_crypto::FieldElement;
+use std::{
+    collections::HashMap,
+    env::var,
+    path::{Path, PathBuf},
+    str::FromStr,
+};
+
+lazy_static! {
+    static ref GENESIS_FOLDER_PATH: PathBuf = Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf().join(".katana");
+    static ref KAKAROT_CONTRACTS_PATH: PathBuf =
+        Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf().join("lib/kakarot/build");
+    static ref INCORRECT_FELT: String = "INCORRECT".to_string();
+    static ref COINBASE_ADDRESS: FieldElement = FieldElement::from(0x12345u32);
+    static ref SALT: FieldElement = FieldElement::from_bytes_be(&[0u8; 32]).expect("Failed to convert salt");
+}
+
+#[serde_as]
+#[derive(Serialize)]
+struct Hex(#[serde_as(as = "UfeHex")] pub FieldElement);
+
+#[derive(Serialize)]
+struct KatanaManifest {
+    declarations: HashMap<String, Hex>,
+    deployments: HashMap<String, Hex>,
+}
+
+fn main() {
+    // Load the env vars.
+    dotenv().ok();
+
+    // Read the env vars.
+    let pk = B256::from_str(&var("EVM_PRIVATE_KEY").expect("Missing EVM private key"))
+        .expect("Failed to parse EVM private key");
+
+    // Read all the classes.
+    let mut builder = KatanaGenesisBuilder::default()
+        .load_classes(KAKAROT_CONTRACTS_PATH.clone())
+        .with_kakarot(*COINBASE_ADDRESS)
+        .expect("Failed to set up Kakarot");
+    builder = builder.with_eoa(pk, None).expect("Failed to set up EOA").fund(pk, U256::from(u128::MAX)).unwrap();
+
+    let cache = builder.cache().clone().into_iter().map(|(k, v)| (k, Hex(v))).collect::<HashMap<_, _>>();
```

Comment:
why is it called cache?

Reply:
just the builder's cache, where I can store some intermediate variables I need during building (e.g. "kakarot_address").

Reply:
it's a cache, could be used for anything, class hashes, addresses, strings in the form of FieldElement